test/ComposerRequireCheckerTest/DefinedSymbolsLocator/LocateDefinedSymbolsFromComposerRuntimeApiTest.php

```diff
@@ -22,19 +23,23 @@ protected function setUp(): void
     }
 
     /** @dataProvider provideComposerJsonWithUnsuitableComposerRuntimeApi */
+    #[DataProvider('provideComposerJsonWithUnsuitableComposerRuntimeApi')]
     public function testNoSymbols(string $composerJson): void
     {
-        $symbols = $this->locate(json_decode($composerJson, true));
-
-        self::assertEmpty($symbols);
+        self::assertEmpty(($this->locator)(Json\typed(
+            $composerJson,
+            JsonLoader::composerDataType(),
+        )));
     }
 
     /** @dataProvider provideComposerJsonWithSuitableComposerRuntimeApi */
+    #[DataProvider('provideComposerJsonWithSuitableComposerRuntimeApi')]
```

Comment:
Do we need both of these comments? Can we remove the docblock tag in favour of the attribute?

Reply:
I kept the old comments due to `psalm/plugin-phpunit` relying on them, but I'm working ti fix the upstream package